styles/settings/font_size_map.scss

```diff
@@ -0,0 +1,24 @@
+// Font Size
+$font-size: ("3xs": 0.813rem,
+"2xs": 0.875rem,
+"1xs": 1rem,
+"sm": 1.063rem,
+"md": 1.125rem,
+"lg": 1.5rem,
+"1xl": 2rem,
+"2xl": 2.5rem,
+"3xl": 3rem,
+"detail-page-name": 3.5rem);
+
+// Font Size (px)
+// $font-size-px: (
+//     '3xs': 13px,
+//     '2xs': 14px,
+//     '1xs': 16px,
+//     'sm': 17px,
+//     'md': 18px,
+//     'lg': 24px,
+//     '1xl': 32px,
+//     '2xl': 40px,
+//     '3xl': 48px,
+//     'detail-page-name': 56px);
```

Comment:
I would rename this file to `_font-sizes.scss`. The leading underscore is a SCSS convention to indicate that the file is meant to be imported. `globals.scss` should be the only SCSS file that doesn't start with an underscore.